src/omv/alloc/fb_alloc.c

```diff
@@ -12,6 +12,12 @@
 #include "framebuffer.h"
 #include "omv_boardconfig.h"
 
+// Use 32-byte alignment on MCUs with no cache (this allows the Cortex-M4 to benefit from DMA buffer alignment).
+#ifndef __DCACHE_PRESENT
+#define __DCACHE_PRESENT 1
+#define __SCB_DCACHE_LINE_SIZE 32
+#endif
+
```

Comment:
If you need to do something later with this flag, ex cache maintenance, you can't force define it like that. I suggest this and just use this value for alignment.
```suggestion
#ifndef __DCACHE_PRESENT
#define FB_ALLOC_ALIGNMENT 32
#else
#define FB_ALLOC_ALIGNMENT  __SCB_DCACHE_LINE_SIZE
#endif

```